pkg/daemon/server/handle.go

```diff
@@ -110,6 +105,22 @@ func (cdh *cniDaemonHandler) handleAdd(req *restful.Request, resp *restful.Respo
 	backOffBase := 5 * time.Microsecond
 	retries := 11
 	ipFamily := ipamtypes.ParseIPFamilyFromString(pod.Annotations[constants.AnnotationIPFamily])
+	handledByWebhook := globalutils.ParseBoolOrDefault(pod.Annotations[constants.AnnotationHandledByWebhook], false)
+
+	if !handledByWebhook {
+		_, _, _, ipFamily, _, err = webhookutils.ParseNetworkConfigOfPodByPriority(context.TODO(), cdh.mgrAPIReader, pod)
+		if err != nil {
+			errMsg := fmt.Errorf("failed to parse network config of pod %v: %v", pod.Name, err)
+			cdh.errorWrapper(errMsg, http.StatusBadRequest, resp)
+			return
+		}
+	}
+
+	if !ipamtypes.IsValidFamilyMode(ipFamily) {
```

Comment:
ip family validated in `PaseNetworkConfigOfPodByPriority`

Reply:
done